cmd/provider/main.go

```diff
@@ -123,6 +124,8 @@ func main() {
 		log.Info("Alpha feature enabled", "flag", features.EnableAlphaManagementPolicies)
 	}
 
+	metrics.SetupMetrics()
```

Comment:
To be consistent with the other setups, can we use `Register` instead of `MustRegister` and pass the error into `kingpin.FataIfError`? The result (a crash) would be the same, but it is more integrated into the control flow.

```suggestion
	kingpin.FatalIfError(metrics.SetupMetrics(), "Cannot setup AWS metrics hook")
```